opik.ps1

```diff
@@ -114,26 +114,42 @@ function Send-InstallReport {
 
     $Timestamp = (Get-Date -Format "yyyy-MM-ddTHH:mm:ssZ")
 
+    $frontendPort = docker inspect -f '{{ (index (index .NetworkSettings.Ports "5173/tcp") 0).HostPort }}' opik-frontend-1 2>$null
+    if (-not $frontendPort) { $frontendPort = 5173 }
+    $uiUrl = "http://localhost:$frontendPort"
+
     if ($EventCompleted -eq "true") {
-        $EventType = "opik_install_completed"
+
+        try {
+            $response = Invoke-WebRequest -Uri "$uiUrl/api/is-alive/ver" -UseBasicParsing -TimeoutSec 2
```

Comment:
@thiagohora can you get version from docker? (without api ping)

Reply:
If we do this, we will get latest for must cases, unless thr user sets the OPIK_VERSION variable